### DISCUSSION TITLE
Error: Call to undefined method Leaf\Http\Response::next()

### DISCUSSION THREAD

**Discussion by thoughtfulviewfinder** [leafsphp/leaf#259] [Q&A]
Hi,

I am trying to implement a JWT verification middleware. The Middleware functions as expected but when I try to introduce the 

`response()->next($token);`

throws an error. I am currently using the following packages.

"leafs/leaf": "^3.5",
 "leafs/auth": "^2.3",
"leafs/db": "^2.1",
"firebase/php-jwt": "^6.10"

Following the documentation mentioned here: [url=https://leafphp.dev/docs/routing/middleware.html](https://leafphp.dev/docs/routing/middleware.html)

**Comment by mychidarko** [comment]
Hi @thoughtfulviewfinder it looks like you need to update your leaf version. You're currently running v3.5, you need to update to either 3.6 or 3.7
